projects/project-player-library/src/lib/pages/details-page/details-page.component.ts

```diff
@@ -11,85 +11,673 @@ import { DailogPopupComponent } from '../../shared/dialog-popup/dailog-popup.com
 export class DetailsPageComponent implements OnInit {
   completedCount: number = 0;
   progressValue: number = 0;
+  actionsList=[
+    {
+      name: "EDIT",
+      icon:"edit"
+    },
+    {
+      name: "SHARE",
+      icon:"ios_share"
+    },
+    {
+      name: "DELETE",
+      icon:"delete"
+    },
+  ];
+
   projectActions = [
     {
       label:"DOWNLOAD",
-      icon1:"cloud_download",
-      emitdata:false
+      icon:"cloud_download",
+      action:false
     },
     {
       label:"SHARE",
-      icon1:"ios_share",
-      emitdata:false
+      icon:"ios_share",
+      action:false
     },
     {
       label:"FILES",
-      icon1:"folder_open",
-      emitdata:false
+      icon:"folder_open",
+      action:false
     },
     {
       label:"SYNC",
-      icon1:"sync",
-      emitdata:false
+      icon:"sync",
+      action:false
     },
   ]
   submitted: boolean = false;
   projectDetails = {
```

Comment:
Create a project-details.compnent.spec.data.ts file and add this JSON data to that file. Import that file here and assign the value to the projectDetails

Reply:
done